js/adapt-contrib-contentObjectTransition.js

```diff
@@ -175,7 +143,6 @@ class ContentObjectTransition extends Backbone.Controller {
     incomingContentObject.$el[0].removeEventListener('transitionend', this.endAnimation);
     incomingContentObject.$el[0].removeEventListener('animationend', this.endAnimation);
 
-    // Use raf to stop any perceptible jumping in IE11
     window.requestAnimationFrame(() => {
```

Comment:
Can this whole piece go if it was IE11 specific?

Reply:
I think the block is needed - without it the animation won't work correctly. Might be that `requestAnimationFrame` was the implementation choice to suit IE11, but it still works for modern browsers. @oliverfoster any comment on this?